src/hooks/useDydxClient.tsx

```diff
@@ -144,7 +152,7 @@ const useDydxClientContext = () => {
   }, [compositeClient, setSelectedGasDenom]);
 
   // ------ Wallet Methods ------ //
-  const getWalletFromEvmSignature = async ({ signature }: { signature: string }) => {
+  const getWalletFromSignature = async ({ signature }: { signature: string }) => {
```

Comment:
[q] i get why we've dropped the `EVM` prefix from signature since now we support solana, but i'm a little confused since we're still using the `onboarding.deriveHDKeyFromEthereumSignature` method - does that just work for solana signatures as well?

Reply:
makes sense, can we go ahead and add a comment there for future reference?

Reply:
✅